common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -213,14 +195,32 @@ trait CommercialSwitches {
   )
 
   val BookLookupSwitch = Switch(
-    SwitchGroup.Commercial,
+    SwitchGroup.CommercialFeeds,
     "book-lookup",
     "If this switch is on, book data will be looked up using a third-party service.",
     safeState = Off,
     sellByDate = never,
     exposeClientSide = false
   )
 
+  val MembersAreaSwitch = Switch(
```

Comment:
I didn't (intentionally) create any new switches - this looks to be an artefact of these switches being removed subsequent to the raising of this PR. I'll merge `master`, which should fix this.